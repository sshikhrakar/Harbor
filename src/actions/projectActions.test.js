import {
  fetchAllProjects,
  fetchAllProjectsErrored,
  fetchAllProjectsFulfilled,
} from './projectActions';

describe('ACTIONS: projectActions', () => {

  it('should create an action to fetch all projects', () => {
    expect(fetchAllProjects()).toMatchSnapshot();
  });

  it('should create an error action for all projects fetched', () => {
    expect(fetchAllProjectsErrored()).toMatchSnapshot();
  });

  it('should create a success action for all projects fetched', () => {
    expect(fetchAllProjectsFulfilled('data')).toMatchSnapshot();
  });

});
