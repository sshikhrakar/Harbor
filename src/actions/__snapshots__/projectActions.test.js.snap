// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ACTIONS: projectActions should create a success action for all projects fetched 1`] = `
Object {
  "projects": "data",
  "type": "PROJECTS/FETCH_ALL_PROJECTS_FULFILLED",
}
`;

exports[`ACTIONS: projectActions should create an action to fetch all projects 1`] = `
Object {
  "type": "PROJECTS/FETCH_ALL_PROJECTS",
}
`;

exports[`ACTIONS: projectActions should create an error action for all projects fetched 1`] = `
Object {
  "type": "PROJECTS/FETCH_ALL_PROJECTS_ERRORED",
}
`;
